docs/ex/modules/module.go

```diff
@@ -26,77 +26,68 @@ import (
 )
 
 // Module is an example of an Fx module's skeleton.
-// region start
-// region provide
-// region invoke
-// region decorate
-// region private
+// --8<-- [start:start]
```

Comment:
This was basically a refactor of the snippet. 
The old version would've worked if we got rid of the duplicates (so we did "provide-1", "provide-2", etc.),
at which point just building them up as needed in the output seemed better.
I also changed the actual resulting code in this case since building these specific cases line by line didn't seem to be adding much.
(This section: https://abhinav.github.io/fx/modules.html#writing-modules)